### DISCUSSION TITLE
Sync two instances of Vaultwarden to have an offsite backup

### DISCUSSION THREAD

**Discussion by mkroemer** [dani-garcia/vaultwarden#1943] [Q&A] [answered]
Hello,
i and a friend of mine are both running a vaultwarden instance on two different homeservers. It would be great to sync these two instances and therefore have a backup if one is not reachable or if the server crashes. Is this possible in any way?
Thanks

**Comment by BlackDex** [comment, marked as answer]
Not really.
There are some ways to run vaultwarden in High Availability with some limitations.

There is no sync between separate servers or what ever.
What you could do, is both have a second vaultwarden running on a different port or something and have both of your databases (and files) synced to that specific container.

How to keep everything in sync is not something vaultwarden can do for you.
One thing i can think of is use something like Nextcloud or Syncthing or something similar.
